### ISSUE TITLE
R: make stan_code, dataset etc more generic

### ISSUE THREAD

**Issue by eerolinna** [MansMeg/posteriordb#29]
Right now we have

```R
po <- posterior("8_schools-8_schools_centered", my_pdb)
dataset(po)
stan_code(po)
```

It would be nice to also have something like
```R
mo <- model("8_schools_centered", my_pdb)
stan_code(mo)
```
and
```R
da <- data("8_schools", my_pdb)
dataset(da)
```

**Comment by eerolinna** [comment]
The python package has generic versions of `stan_code`, `dataset` etc

```python
po = Posterior("8_schools-8_schools_centered", my_pdb)
po.dataset()
po.stan_code()

mo = Model("8_schools_centered", my_pdb)
mo.stan_code()

da = Data("8_schools", my_pdb)
da.dataset()
```

I think it would be good to have them in the R version too.

Lets say the user only wants to find the stan code for  `blr`. Currently in R they first need to find some posterior that has `blr` as the model and only then they can access the stan code. If `stan_code` was a generic function then it would not be necessary to search for a posterior and they could just directly obtain the stan code from the `blr` model.

**Comment by MansMeg** [comment]
I agree!

**Comment by eerolinna** [comment]
So this is going to happen? Or was this closed as this is kind of same as #105?

**Comment by MansMeg** [comment]
It has already been fixed. Should have mentioned that.
